docs/conf.py

```diff
@@ -98,7 +98,7 @@
     "github_url": "https://github.com/MideTechnology/endaq-python",
     "twitter_url": "https://twitter.com/enDAQ_sensors",
     "collapse_navigation": True,
-    #"external_links": [{"name": "Learn", "url": "https://numpy.org/numpy-tutorials/"}],
+    "google_analytics_id": "G-E9QXH4H5LP",
```

Comment:
Is this the correct key? I looked at the [Sphinx docs](https://sphinx-rtd-theme.readthedocs.io/en/stable/configuring.html#confval-analytics_id) and only saw reference to `"analytics_id"`.

Reply:
It may be Read The Docs only `https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/configuring.html#google-analytics`